customer.rb

```diff
@@ -1 +1,44 @@
-# See README.md
+class Customer
+  @@all = []
+  attr_accessor :id, :first_name, :last_name, :location, :orders, :money_spent, :amount_of_orders
+  def initialize(id, first_name, last_name, location)
+    @id = id
+    @first_name = first_name
+    @last_name = last_name
+    @location = location
+    @orders = []
+    @items = [] 
+    @prices = []
+    @money_spent
+    @@all.push(self)
+    @amount_of_orders
+  end
+  def order_to_customer(order)
```

Comment:
I would rename it. `order_to_customer` sounds a bit like you want to parse an order object into a customer object or something like this.

Maybe, but also not the best naming:
```suggestion
  def add_order_by_id(order)
```

Reply:
changed it to assign_order_to_customer, as I want to be very clear for myself what is happening, hope that's okay :)

Reply:
of course, the more descriptive the better in most cases 😊